chains/orchestrator-relays/node/tanssi-relay-service/src/chain_spec.rs

```diff
@@ -213,46 +211,6 @@ pub fn dancelight_development_config(
     .build())
 }
 
-/// Dancelight local testnet config (multivalidator Alice + Bob)
-#[cfg(feature = "dancelight-native")]
-pub fn dancelight_local_testnet_config(
-    container_chains: Vec<String>,
-    mock_container_chains: Vec<ParaId>,
-    invulnerables: Vec<String>,
-) -> Result<DancelightChainSpec, String> {
-    let container_chains: Vec<_> = container_chains
-        .iter()
-        .map(|x| {
-            container_chain_genesis_data_from_path(x).unwrap_or_else(|e| {
-                panic!(
-                    "Failed to build genesis data for container chain {:?}: {}",
-                    x, e
-                )
-            })
-        })
-        .chain(
-            mock_container_chains
-                .iter()
-                .map(|x| (*x, mock_container_chain_genesis_data(*x), vec![])),
-        )
-        .collect();
-
-    Ok(DancelightChainSpec::builder(
-        dancelight::fast_runtime_binary::WASM_BINARY
```

Comment:
Try to keep this function but use the non fast runtime binary here. For zombienet tests we compile with --features=fast-runtime, so if this is implemented correctly, it will be a fast runtime even if the name doesn't say it.